projects/step-frontend/src/lib/modules/_common/components/login/login.component.scss

```diff
@@ -1,31 +1,113 @@
-mat-form-field {
+:host {
   width: 100%;
-
-  ::ng-deep .mat-form-field-flex {
-    background: none !important;
-  }
-
-  ::ng-deep input {
-    &:-webkit-autofill,
-    &:-webkit-autofill:hover,
-    &:-webkit-autofill:focus,
-    &:-webkit-autofill:active {
-      transition: background-color 5000s ease-in-out 0s;
-    }
-  }
+  display: flex;
 }
 
-:host {
-  width: 100%;
+.step-login-flex-child {
+  flex: 1;
   display: flex;
   flex-wrap: nowrap;
   justify-content: center;
   flex-direction: row;
   align-items: center;
+
+  .step-login-content {
+    width: 360px;
+    margin-top: -20vh;
+
+    .step-login-logo {
+      margin: 0 auto 30px;
+      width: 120px;
+
+      img {
+        max-width: 120px;
+        margin: 0 auto;
+      }
+    }
+
+    .step-login-heading-text {
+      text-align: center;
+      font-size: 28px;
+      font-weight: 600;
+      line-height: 32px;
+    }
+
+    .step-login-heading-text-additional-margin {
+      margin-bottom: 40px;
+    }
+
+    .step-login-description-text {
+      text-align: center;
+      font-size: 16px;
+      line-height: 24px;
+      color: grey;
+      margin-bottom: 30px;
+    }
+
+    .step-login-input-wrapper {
+      margin-top: 25px;
+
+      .step-login-input-label {
+        font-weight: 500;
+      }
+
+      input {
+        width: 100%;
+        border-style: solid;
+        border-color: lightgray;
+        border-width: thin;
+        border-radius: 8px;
+        font-size: 16px;
+        font-weight: 400;
+        padding: 12px;
+      }
+
+      input:focus {
+        outline: none !important;
+        border-color: #0081caff;
+        border-width: thin;
+      }
+    }
+
+    .step-login-forgot-password-text {
+      font-weight: 500;
+      margin-top: 30px;
+      margin-bottom: -10px;
+      color: #006299;
+    }
+
+    .step-login-button {
+      margin-top: 40px;
+      height: 44px;
+      color: white;
+      border-radius: 8px;
+      background-color: #0081caff;
```

Comment:
@neogucky Do we need to control the colors of this page by theme?

Reply:
I'm not sure why this comment is here and not in the other box. But colours should be managed by the theme already here. Maybe give @chillbirdo advice how to do this cleanly.

Reply:
done! thx @dvladir for the help

Reply:
instead of the exact colors from the figma, I am using now the nearest colors defined in the theme